packages/back-end/src/models/FeatureModel.ts

```diff
@@ -142,12 +144,14 @@ async function logFeatureUpdatedEvent(
   previous: FeatureInterface,
   current: FeatureInterface
 ): Promise<string> {
+  const savedGroupMap = await getSavedGroupMap(organization);
```

Comment:
The change to log `ApiFeature` instead of the internal `FeatureInterface` requires us to make this saved group call in all our `logFeature...` calls.